README.md

```diff
@@ -5,8 +5,13 @@ All you should need is a `bot.py` config file that looks like this:
 ```
 CLIENT_ID=''
 CLIENT_SECRET=''
+DISCORD_TOKEN=''
+FOOTER=''
```

Comment:
No need for this here, this isn't a configuration related constant, so it should remain in the code itself.